lib/client-assets.php

```diff
@@ -87,8 +87,6 @@ function register_tinymce_scripts() {
 			gutenberg_override_script( 'wp-tinymce-root', includes_url( 'js/tinymce/' ) . "tinymce{$mce_suffix}.js", array(), $tinymce_version );
 			gutenberg_override_script( 'wp-tinymce', includes_url( 'js/tinymce/' ) . "plugins/compat3x/plugin{$suffix}.js", array( 'wp-tinymce-root' ), $tinymce_version );
 		}
-
-		gutenberg_override_script( 'wp-tinymce-lists', includes_url( 'js/tinymce/' ) . "plugins/lists/plugin{$suffix}.js", array( 'wp-tinymce' ), $tinymce_version );
```

Comment:
FYI: This line will separately become removed in #13466.

Reply:
What's the process of removing core PHP related to Gutenberg PRs?

Reply:
> What's the process of removing core PHP related to Gutenberg PRs?

I'm currently gutting most of the PHP, so that we'll get to a point where Gutenberg PHP is primarily just overrides to the default core scripts and block implementations.

See also:

- https://github.com/WordPress/gutenberg/projects/23
- https://github.com/WordPress/gutenberg/issues/11015